qa/crontab/teuthology-cronjobs

```diff
@@ -88,7 +88,7 @@ CEPH_QA_EMAIL="ceph-qa@lists.ceph.com"
 ## upgrades suites for on luminous
 ## !!!! three suites below MUST use --suite-branch hammer, kraken OR jewel
 ## --filter "ubuntu_14.04,ubuntu_16.04,centos_7.4" == test only supported distro
-45 05 * * 7  CEPH_BRANCH=luminous; MACHINE_NAME=smithi;   /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -m $MACHINE_NAME -s upgrade/client-upgrade-hammer -k distro -e $CEPH_QA_EMAIL --suite-branch hammer
+45 05 * * 7  CEPH_BRANCH=luminous; MACHINE_NAME=smithi;   /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -m $MACHINE_NAME -s upgrade/client-upgrade-hammer -k distro -e $CEPH_QA_EMAIL --suite-branch hammer "ubuntu_14.04"
```

Comment:
Doesn't this need ```--filter```?

Reply:
@dillaman of cause it does :) (missed to ad change before committing), the tests ran correctly tho